src/utils/jwt.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/no-namespace */
+import jwt from 'jsonwebtoken'
+
```

Comment:
It would be better to have common place for having common environments
```suggestion
import {JWT_SECRET, JWT_EXPIRE_TIME} from '../config'; 
```
and in `config/index.ts`
```ts
export const JWT_SECRET=process.env.JWT_SECRET??'USER-AUTH'
export const JWT_EXPIRE_TIME=process.env.JWT_EXPIRE_TIME??'7d'
```
this make sure that whenever you need to change name  for env you don't have to change allover other place